src/modules/linagora.esn.account/frontend/js/services.js

```diff
@@ -1,76 +1,83 @@
-    httpConfigurer.manageRestangular(restangularInstance, '/account/api');
+  angular.module('linagora.esn.account')
 
-    return restangularInstance;
-  })
+    .factory('AccountRestangular', function(Restangular, httpConfigurer) {
+      var restangularInstance = Restangular.withConfig(function(RestangularConfigurer) {
+        RestangularConfigurer.setFullResponse(true);
+      });
 
-  .factory('accountService', function(AccountRestangular) {
+      httpConfigurer.manageRestangular(restangularInstance, '/account/api');
 
-    function getAccounts(options) {
-      return AccountRestangular.all('accounts').getList(options);
-    }
+      return restangularInstance;
+    })
 
-    function getAccountProviders() {
-      return AccountRestangular.all('accounts').all('providers').getList();
-    }
+    .factory('accountService', function(AccountRestangular) {
 
-    return {
-      getAccounts: getAccounts,
-      getAccountProviders: getAccountProviders
-    };
+      function getAccounts(options) {
+        return AccountRestangular.all('accounts').getList(options);
+      }
 
-  })
+      function getAccountProviders() {
+        return AccountRestangular.all('accounts').all('providers').getList();
+      }
 
-  .factory('accountMessageRegistry', function(OAUTH_DEFAULT_MESSAGES, OAUTH_UNKNOWN_MESSAGE) {
+      return {
+        getAccounts: getAccounts,
+        getAccountProviders: getAccountProviders
+      };
 
-    var cache = {};
+    })
 
-    function register(provider, messages) {
-      cache[provider] = messages;
-    }
+    .factory('accountMessageRegistry', function(OAUTH_DEFAULT_MESSAGES, OAUTH_UNKNOWN_MESSAGE) {
 
-    function get(provider, type) {
-      if (cache[provider]) {
-        var messages = cache[provider];
+      var cache = {};
 
-        return messages[type] ? messages[type] : OAUTH_DEFAULT_MESSAGES[type] || OAUTH_UNKNOWN_MESSAGE;
+      function register(provider, messages) {
+        cache[provider] = messages;
       }
 
-      return OAUTH_DEFAULT_MESSAGES[type] || OAUTH_UNKNOWN_MESSAGE;
-    }
+      function get(provider, type) {
+        if (cache[provider]) {
+          var messages = cache[provider];
+
+          return messages[type] ? messages[type] : OAUTH_DEFAULT_MESSAGES[type] || OAUTH_UNKNOWN_MESSAGE;
+        }
 
-    return {
-      register: register,
-      get: get
-    };
-  })
+        return OAUTH_DEFAULT_MESSAGES[type] || OAUTH_UNKNOWN_MESSAGE;
+      }
 
-  .factory('displayAccountMessage', function(accountMessageRegistry, notificationFactory) {
-    return function(provider, status) {
-      notificationFactory.weakInfo('', accountMessageRegistry.get(provider, status));
-    };
-  })
+      return {
+        register: register,
+        get: get
+      };
+    })
 
-  .factory('socialHelper', function(OAUTH_SOCIAL_MESSAGES, _) {
-    function getAccountMessages(type) {
-      var message = {};
+    .factory('displayAccountMessage', function(accountMessageRegistry, notificationFactory) {
```

Comment:
missing require for the dependencies injections

Reply:
in root module